dataikuapi/dss/projectdeployer.py

```diff
@@ -461,6 +520,20 @@ def __init__(self, client, project_key, light_status):
         self.project_key = project_key
         self.light_status = light_status
 
+    def list_deployments(self, infra_id=None):
+        """
+        Returns the deployments that have been created from this published project
+
+        :param str infra_id: Identifier of an infra, allows to only keep in the returned list the deployments on this infra.
+        If not set, the list contains all the deployments using this published project, across every infra of the Project Deployer.
+
+        :returns: a list of deployments
+        :rtype: list of :class:`dataikuapi.dss.projectdeployer.DSSProjectDeployerDeployment`
+        """
+        if infra_id is None:
+            return [DSSProjectDeployerDeployment(self.client, deployment.id) for deployment in self.light_status["deployments"]]
+        return [DSSProjectDeployerDeployment(self.client, deployment.id) for deployment in self.light_status["deployments"] if infra_id == deployment.infraId]
```

Comment:
Same `deployment["id"]` thing in these two lines.

Reply:
`deployment["infraId"]` as well 😅

Reply:
:shame: